tools/packaging/static-build/virtiofsd/build-static-virtiofsd.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ARCH=$(uname -m)
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../../scripts/lib.sh"
+
+config_dir="${script_dir}/../../scripts/"
+
+virtiofsd_repo="${virtiofsd_repo:-}"
+virtiofsd_dir="virtiofsd"
+virtiofsd_version="${virtiofsd_version:-}"
+kata_version="${kata_version:-}"
+
+if [ -z "$virtiofsd_repo" ]; then
+	info "Get virtiofsd information from runtime versions.yaml"
+        virtiofsd_url=$(get_from_kata_deps "externals.virtiofsd.url" "${kata_version}")
+	[ -n "$virtiofsd_url" ] || die "failed to get virtiofsd url"
+        virtiofsd_repo="${virtiofsd_url}.git"
+fi
+[ -n "$virtiofsd_repo" ] || die "failed to get virtiofsd repo"
```

Comment:
I don't see `virtiofsd_repo` actually used - is that intentional?  If it's a preparation for archs that will use it, it might be cleaner to put the `virtiofsd_repo` code into a function and only execute it when it's needed. (?)

Reply:
Updated!